### PR TITLE
[id] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@itsfaqih can you check this pull request, please ? thank you

File: locales/id/id.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "Barbados": "Barbados",
     "Before proceeding, please check your email for a verification link.": "Sebelum melanjutkan, silahkan periksa surel Anda untuk tautan verifikasi.",
     "Belarus": "Belarus",
-    "Belgium": "Belgia",
+    "Belgium": "Belgianame",
```
Comment (itsfaqih) [inline locales/id/id.json:78]: It's Belgia

File: locales/id/id.json [hunk 2]
```diff
@@ -171,9 +171,9 @@
     "E-Mail Address": "Alamat Surel",
     "Ecuador": "Ekuador",
     "Edit": "Edit",
```
Comment (itsfaqih) [inline locales/id/id.json:173]: I just realized even though both "edit" and "sunting" are valid in Indonesian, I guess "sunting" is more appropriate here

File: locales/id/id.json [hunk 3]
```diff
@@ -171,9 +171,9 @@
...
-    "Edit :resource": "Edit :resource",
+    "Edit :resource": "Sunting :resource",
     "Edit Attached": "Edit Lampiran",
-    "Editor": "Editor",
+    "Editor": "Penyunting",
     "Editor users have the ability to read, create, and update.": "Editor memiliki kemampuan untuk membaca, membuat, dan memperbarui.",
```
Comment (itsfaqih) [inline locales/id/id.json:177]: Change "Editor" to "Penyunting"

File: locales/id/id.json [hunk 4]
```diff
@@ -431,18 +431,18 @@
     "Restore Selected": "Pulihkan Yang Dipilih",
     "results": "hasil",
     "Reunion": "Reuni",
-    "Role": "Peran",
+    "Role": "Tugas",
```
Comment (itsfaqih) [inline locales/id/id.json:434]: In this one, "Peran" is more fit than "Tugas". "Tugas" could also mean "Task" while the only translation of "Peran" in English is "Role"

File: locales/id/id.json [hunk 5]
```diff
@@ -431,18 +431,18 @@
...
     "Romania": "Romania",
     "Run Action": "Jalankan Aksi",
     "Russian Federation": "Federasi Rusia",
     "Rwanda": "Rwanda",
     "Saint Barthelemy": "Saint Barthelemy",
     "Saint Helena": "Saint Helena",
-    "Saint Kitts And Nevis": "Saint Kitts dan Nevis",
+    "Saint Kitts And Nevis": "Saint Kitts And Nevis",
```
Comment (itsfaqih) [inline locales/id/id.json:441]: "dan" means "and', may I know the reason why changing it back to "and"?

File: locales/id/id.json [hunk 6]
```diff
@@ -529,7 +529,7 @@
     "There are no available options for this resource.": "Tidak ada opsi yang tersedia untuk sumber ini.",
     "There was a problem executing the action.": "Terdapat masalah saat menjalankan aksi.",
     "There was a problem submitting the form.": "Terdapat masalah saat mengirim form.",
-    "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Orang-orang tersebut telah diundang ke tim Anda dan sudah dikirim email undangan. Mereka bisa bergabung ke dalam tim dengan menerima email undangan.",
+    "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Orang-orang ini telah diundang ke tim Anda dan telah dikirim email. Mereka mungkin bergabung dengan tim dengan menerima email undangan.",
```
Comment (itsfaqih) [inline locales/id/id.json:532]: Both "tersebut" or "ini" are fine. But, for the "may", in this context, "bisa" or "dapat" (which can also means "could") is more fitting than "mungkin" (which means "probably")

**Review by itsfaqih — CHANGES_REQUESTED** [review]
I think that's all. Let me know if you need any help related to this translation

**Comment by caouecs** [comment]
Ok, I merge and I fix

**Comment by caouecs** [comment]
done

**Comment by andrey-helldar** [comment]
I haven't made any changes to this PR yet ><

**Comment by caouecs** [comment]
I fixed the translations

**Pull request merged**